plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -963,11 +963,14 @@ class FossId internal constructor(
 
         val pendingFilesCount = (rawResults.listPendingFiles - newlyMarkedFiles.toSet()).size
 
+        val fossIdScanUrl = buildFossIdScanUrl(config.serverUrl, result.scanCode)
+
         issues.add(
             0,
             Issue(
                 source = descriptor.id,
-                message = "This scan has $pendingFilesCount file(s) pending identification in FossID.",
+                message = "This scan has $pendingFilesCount file(s) pending identification in FossID. " +
+                    "Please review and resolve them on the [FossID Website]($fossIdScanUrl).",
```

Comment:
Issue also has `howToFix`, maybe this is a better fit for the added information? What do you think @oss-review-toolkit/core-devs ?

Reply:
> Issue also has `howToFix`

I don't think there's `howToFix. Do you mean `affectedPath`?